eth-geth/templates/secret.yaml

```diff
@@ -0,0 +1,21 @@
+---
+{{ if .Values.key }}
```

Comment:
Should this be `Values.password` and vice-versa?

Reply:
You're right, my bad. Thank you